OpenRA.Mods.Common/Widgets/Logic/ServerListLogic.cs

```diff
@@ -367,9 +367,11 @@ public void RefreshServerList()
 						}
 					}
 				}
-				catch
+				catch (Exception e)
 				{
 					searchStatus = SearchStatus.Failed;
+					Log.Write("debug", "Server list query failed. Error was:");
+					Log.Write("debug", e.StackTrace);
```

Comment:
This reports the stack trace, but not the exception type!

Can we please change this to match the more standard logging pattern we use in may other places?
```csharp
Log.Write("debug", "Failed to query server list with exception: {0}", e);
```

Reply:
That was unintentional.